.gitignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
`yarn` and `npm` share the same responsibility. Something wired will happened if `yarn.lock` and `package-lock.json` exist at the same time.  Please commit one of them instead of both.

`yarn.lock` and ``